### ISSUE TITLE
Better code commenting?

### ISSUE THREAD

**Issue by karan** [jawerty/AlienFeed#11]
I think it would be a good idea to comment the code well before we move into more development. Just need to make sure maintenance is easy.

**Comment by jawerty** [comment]
Yeah, I'm adding better comments to the next version 0.3.2. Thanks for the idea.

**Comment by karan** [comment]
To all contributors, please document the code as much as possible. It's very hard to read through the code when many people work, use different naming conventions and philosophies. Please add comments to almost anything you write.
